shared/js/smart-screen/simple_key_navigation.js

```diff
@@ -30,7 +32,29 @@
   };
 
   proto.stop = function skn_stop() {
-    this.target.removeEventListener('keydown', this);
```

Comment:
I suggest not to add your code here since "start" and "stop" are reserved module method names in system app while "stop" method means reversing all what you have done in "start" method.

Try to use "pause / resume" or "setEnable(true / false)" instead.